app/models/katello/root_repository.rb

```diff
@@ -455,6 +456,22 @@ def format_arches
       end
     end
 
+    def allow_updates?(additional_allowed_fields=[])          
+      # allow updates for non-container-push repos, repos not in default view, and
+      # repos with a library instance
+      return true unless is_container_push && library_instance.present?
+
+      # let updates that contain ONLY allowed strings through
+      allowed_fields = ['updated_at', 'content_id'] + additional_allowed_fields
```

Comment:
Probably better to put these in a constant

Reply:
I had the same thought but ruby was very mad with the syntax as-is. It said I couldn't use this data structure specifically. I'll see if there's another way to achieve the result.

Reply:
hm, what specific cop was it? Often the cop documentation has examples of how to do it correctly.

Reply:
The only failing cop I see is around whitespace:
```
app/models/katello/root_repository.rb:459:49: C: Layout/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.
    def allow_updates?(additional_allowed_fields=[])          
                                                ^
app/models/katello/root_repository.rb:459:53: C: Layout/TrailingWhitespace: Trailing whitespace detected.
    def allow_updates?(additional_allowed_fields=[])  
```
looks like you just need to change it to `additional_allowed_fields = []`

Reply:
Thanks Jeremy :)

The newest push includes the constants (implemented correctly this time) and fixes my Rubocop errors.